src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -106,6 +107,8 @@ const Sidebar = ({ drawerOpen, setDrawerOpen, moveContent, setMoveContent }: Sid
     setMoveContent(!moveContent);
   };
 
+  const { data: organization } = useCurrentOrganization();
```

Comment:
You should add a check for if data is null it should put the loading indicator and same for error message